types/firefox-webext-browser/index.d.ts

```diff
@@ -697,6 +700,8 @@ declare namespace browser._manifest {
 
     type _MatchPattern = "<all_urls>";
 
+    type _PersistentBackgroundProperty = "undefined" | "undefined";
```

Comment:
Uh, why is this duplicated?

Reply:
Not only was that duplicated, it was also totally wrong.. The generator really didn't like the new manifest it seems. Should be rolled back to a version that actually makes sense now.